src/UsersModule/Components/UsersList/UsersList.tsx

```diff
@@ -1,7 +1,260 @@
+                            </button>
+                            <ul
+                              className={`${styles.dropBtn} dropdown-menu text-black `}
+                            >
+                              <li className="position-relative">
+                                <span
+                                  className={`${styles.triangle} position-absolute `}
+                                >
+                                  <i className="border-1  h2 fa-solid fa-caret-up"></i>
+                                </span>
+                              </li>
+                              <li>
+                                {" "}
+                                <button
+                                  onClick={() => {
+                                    setUserDetails(user);
+                                    handleShowView();
+                                  }}
+                                  className="dropdown-item "
+                                  type="button"
+                                >
+                                  <i className="fa-solid fa-eye me-1 text-success"></i>
+                                  View
+                                </button>
+                              </li>
+                              <li>
+                                <button
+                                  onClick={() => {
+                                    setUsersId(user.id);
+                                    setIsActivatedUser(user.isActivated);
+                                    handleShowToggle();
+                                  }}
+                                  className="dropdown-item "
+                                  type="button"
+                                >
+                                  {user.isActivated ? (
+                                    <i className="text-danger fa-solid fa-toggle-off me-1"></i>
+                                  ) : (
+                                    <i className="h5 text-success fa-solid fa-toggle-on me-1"></i>
+                                  )}
+                                  Toggle
+                                </button>
+                              </li>
+                            </ul>
+                          </div>
+                        </td>
+                      </tr>
+                    ))}
+                  </tbody>
+                  <tfoot>
+                    <tr>
+                     
+                      <td colSpan={10} >
+                      <Pagination pageNum={pageNum} setPageNum={setPageNum} pagesArray={pagesArray}/>
+                      </td>
+                    </tr>
+                  </tfoot>
+                </table>
+              </div>
+            ) : (
+              "d"
+            )}{" "}
+          </>
+        ) : (
+          <div className=" pt-5 mt-5 ">
+            <Loading components={1} />
+          </div>
+        )}
+      </div>
+
+      <>
+        <Modal
```

Comment:
We can take this modal into a different component say (ViewModal) and get it here to make the code easier to follow.